### ISSUE TITLE
likes recived

### ISSUE THREAD

**Issue by binayaksadangi** [benawad/vsinder#187]
user should be able to  see who liked their profile

**Comment by unDeadHerbs** [comment]
I don't think this would be a good idea for two reasons.  Firstly, it makes liking someone's profile a much more active social action, since it actively changes other's view; whereas, currently, it's the liking of an other that changes one's own view.  Secondly, it allows people to be creepier by watching the profiles that have liked them and selectively liking them back, particularly if #20 gets added, encouraging reactivate usage of the app over proactive.

**Comment by FotieMConstant** [comment]
I think @unDeadHerbs has a point!

**Comment by unDeadHerbs** [comment]
Until further discussion.
